app/styles/modules/_custom-rows.scss

```diff
@@ -86,6 +86,16 @@ ul.links {
 }
 
 .choose-what-to-sync-row {
+  &:active{
+    .fxa-checkbox__box-outline{
+      border: 2px solid $checkbox-border-color;
+
+      .fxa-checkbox__tick-outline{
+        background: url("/images/tick.svg");
```

Comment:
Sounds good, created [Issue](https://github.com/mozilla/fxa-checkbox/issues/4) to track this.